server/src/models/group.rs

```diff
@@ -0,0 +1,34 @@
+use chrono::prelude::*;
+use diesel::prelude::*;
+
+use serde::{Deserialize, Insertable, Queryable, Selectable, Serialize};
+
+use create::schema::groups;
+
+#[derive(Serialize, Selectable, Queryable)]
+pub struct Group {
+    id: i32,
+    owner_id: i32,
+    created_at: DateTime<Utc>,
+}
+
+#[derive(Deserialize, Insertable)]
+#[diesel(table_name = groups)]
+pub struct NewGroup {
+    owner_id: i32,
+}
+
+#[derive(Serialize, Selectable, Queryable)]
+pub struct UserGroup {
+    id: i32,
+    user_id: i32,
+    group_id: i32,
+    created_at: DateTime<Utc>,
+}
+
+#[derive(Serialize, Insertable)]
+#[diesel(table_name = user_groups)]
+pub struct NewUserGroup {
+    user_id: i32,
+    group_id: i32,
```

Comment:
You don't need `group_id` to create a user group. This id should come from database.